lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -3218,8 +3218,12 @@ public String getReleaseString(SimplePageItem i) {
 		     return null;
 		 if (page.isHidden())
 		     return messageLocator.getMessage("simplepage.hiddenpage");
-		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date()))
-		     return messageLocator.getMessage("simplepage.pagenotreleased");
+		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date())) {
+			 Date releaseDate = page.getReleaseDate();
+			 SimpleDateFormat dateFormat = new SimpleDateFormat("EEE M/d/yy");
```

Comment:
Fair point @ottenhoff. I've updated the pull request and the date should be internationalized now. I'm happy to change the format, message, or combine it into a DateTime instance if that is preferred.

I went ahead and changed the method header for getReleaseString since it was only called in one place (ShowPageProducer) but I can make a new one and keep the original with just the one parameter if that's preferred.